ui/app/controllers/fusor_ui/concerns/application_controller_extensions.rb

```diff
@@ -0,0 +1,14 @@
+module FusorUi
+  module Concerns
+    module ApplicationControllerExtensions
+      extend ActiveSupport::Concern
+
+      included do
+        def welcome
+          redirect_to new_fusor_deployment_path
+        end
```

Comment:
@jmrodri,correct, the `welcome` method always runs, but the concern / mixin / extension overrode it. In your code, you didn't overwrite `weclome`. Notice, in most cases, it's good to call `super` in the mixin, but in my case, I didn't want to.